crates/bevy_pbr/src/render/light.rs

```diff
@@ -216,6 +216,12 @@ pub struct GpuLights {
 
 // NOTE: this must be kept in sync with the same constants in pbr.frag
 pub const MAX_UNIFORM_BUFFER_POINT_LIGHTS: usize = 256;
+
+//NOTE: When running bevy on Adreno GPU chipsets in WebGL, any value above 1 will result in a crash
+// when loading the wgsl "pbr_functions.wgsl" in the function apply_fog.
+#[cfg(feature = "webgl")]
+pub const MAX_DIRECTIONAL_LIGHTS: usize = 1;
+#[cfg(not(feature = "webgl"))]
```

Comment:
with how webgl is now set up, this needs to check too that it's not building for wasm

```suggestion
#[cfg(all(feature = "webgl", target_arch = "wasm32"))]
pub const MAX_DIRECTIONAL_LIGHTS: usize = 1;
#[cfg(any(not(feature = "webgl"), not(target_arch = "wasm32")))]
```

Reply:
as we want to keep webgl2 as the default wasm platform and there are no per platform default features, the webgl feature is now enabled by default on all platforms

Reply:
Nice catch!